vector/src/main/res/layout/item_room_list_empty.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<im.vector.app.core.platform.StateView xmlns:android="http://schemas.android.com/apk/res/android"
+    android:id="@+id/stateView"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:background="?android:colorBackground">
+
+</im.vector.app.core.platform.StateView>
```

Comment:
If we decide to keep this impl, I would rename the layout to `item_state_view.xml`

Reply:
thanks